integrated guess

reedchen19 opened the pull request:
added .action file, server, and python script for guessing the location of the pinger with the cheap set of hydrophones

pep8speaks commented:
Hello @reedchen19! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py`](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py):

> [Line 3:1](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L3): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 17:1](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L17): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 40:9](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L40): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 46:9](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L46): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 47:9](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L47): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 56:80](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L56): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (116 > 79 characters)
> [Line 59:9](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L59): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 66:9](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L66): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 71:9](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L71): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 75:1](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L75): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1
> [Line 77:80](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L77): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (115 > 79 characters)
> [Line 79:1](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L79): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1
> [Line 86:1](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L86): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1
> [Line 105:80](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/cheap_cross_cor.py#L105): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (82 > 79 characters)

* In the file [`onboard/catkin_ws/src/acoustics/scripts/guess_server.py`](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/guess_server.py):

> [Line 5:80](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/guess_server.py#L5): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (94 > 79 characters)
> [Line 16:80](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/guess_server.py#L16): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (111 > 79 characters)
> [Line 34:80](https://github.com/DukeRobotics/robosub-ros/blob/792933bce6eaebca7f217b30362c4db3a8ecf1ca/onboard/catkin_ws/src/acoustics/scripts/guess_server.py#L34): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (96 > 79 characters)

reedchen19 commented:
Thanks for the suggestions! I followed all of them and changed the correlation script to use list comprehension. I moved the filtering and plotting functions to acoustics_math, and changed the feedback to use uint32 curr_stage and total_stage.
